lib/build-tokens.js

```diff
@@ -91,45 +112,62 @@ async function buildTokensCommand(commandArgs) {
         transform: (token) => colorTransform(token, themeVariant),
       },
     },
-    format: {
-      'css/custom-variables': formatterArgs => createCustomCSSVariables({
-        formatterArgs,
-        themeVariant,
-      }),
-    },
     platforms: {
       css: {
         ...coreConfig.platforms.css,
         files: [
           {
             format: 'css/custom-variables',
             destination: `themes/${themeVariant}/variables.css`,
-            filter: hasSourceTokensOnly ? 'isSource' : undefined,
+            filter: hasSourceTokensOnly
+              ? `isSource.${themeVariant}`
+              : `isThemeVariant.${themeVariant}`,
             options: {
-              outputReferences: !hasSourceTokensOnly,
+              outputReferences,
+              formatting: {
+                fileHeaderTimestamp: true,
+              },
```

Comment:
I see this block now exists in a couple spots in this file. It's only 3 lines so I'm not *worried* about the copypasta, but I am curious as to if there's a "built-in" way to have a "global" formatting setting. If there isn't a "built-in" way I'm thinking it might be cool to move this out to a const like we have with `defaultParams` so if anyone wants to change the formatting they can do it in one place.

Reply:
@adamstankiewicz I think this is the last lingering question I have before giving this a :heavy_check_mark:!

Reply:
Plan to investigate later this afternoon. I do recall seeing somewhere in the style-dictionary docs about global settings, so it might be possible. If the global approach isn't possible, I agree abstracting out a `defaultParams` or similar would be worthwhile to be more DRY. I'll share an update later :)

Reply:
Just pushed up a commit to re-introduce a custom file header, and apply it globally across all files in the `css` platform.

```css
/**
 * Do not edit directly, this file was auto-generated. while transforming design tokens.
 * See <root>/tokens/README.md for more details.
 */
```